mu4/notation/inotationinteraction.h

```diff
@@ -53,7 +53,8 @@ class INotationInteraction
     // Select
     virtual Element* hitElement(const QPointF& pos, float width) const = 0;
     virtual int hitStaffIndex(const QPointF& pos) const = 0;
-    virtual void select(Element* element, SelectType type, int staffIndex = 0) = 0;
+    virtual void select(Element* e, SelectType type, int staffIdx = 0) = 0;
+    virtual void select(std::vector<Element*> elements, SelectType type, int staffIndex = 0) = 0;
```

Comment:
Better to unite select(Element* e, ...) and select(std::vector<Element*> elements, ...)